evansmusomi/react-native-102
# Switches Auth Modes on Auth Screen

*What this does*
- Enables user to switch between login and signup form on Auth screen

*How to verify*
- The login form is displayed by default, it doesn't include a confirm password field
- When on login auth mode you should be able to submit after filling in the email and password
- Pressing the "switch to signup" button should reveal a confirm password field
- When on the sign up auth mode you should fill in confirm password to be able to submit
